.github/workflows/python-ci.yml

```diff
@@ -49,8 +49,11 @@ jobs:
         env:
           CI: true
         run: |
-          ./mvnw clean package -DskipTests -P scala-${{ matrix.spark-version.scala }} --no-transfer-progress
+          ./maven-release.sh package ${{ matrix.spark-version.scala }}
+          head -n 15 spark-3/pom.xml
       - name: Run tests for Spark ${{ matrix.spark-version.ext }} and Neo4j ${{ matrix.neo4j-version }}
         run: |
+          pwd
+          head -n 15 spark-3/pom.xml
```

Comment:
@fbiville Do we need this?

Reply:
nope, will remove it now that it finally works